tests/baselines/reference/user/jimp.log

```diff
@@ -0,0 +1,11 @@
+Exit Code: 1
+Standard output:
+node_modules/@jimp/core/index.d.ts(200,3): error TS2411: Property 'class' of type 'undefined' is not assignable to string index type 'Function'.
```

Comment:
It's funny, because I added `jimp` to our user suite, because I'd had issues in the past where they shipped broken types. And it looks like they are again~ Someone'a already working on it: https://github.com/oliver-moran/jimp/pull/786

Reply:
That'd be me - apologies